daisy_workflows/image_test/metadata-ssh-tester.py

```diff
@@ -14,205 +14,99 @@
 # limitations under the License.
 
 
-import logging
-import re
-import time
-import uuid
-
 import utils
 
 utils.AptGetInstall(['python-pip'])
 utils.Execute(['pip', 'install', '--upgrade', 'google-api-python-client'])
 
 from googleapiclient import discovery
 from oauth2client.client import GoogleCredentials
```

Comment:
From our style guide - use imports for packages and modules only, not for individual classes or functions.